README.md

```diff
@@ -102,15 +102,15 @@ Example:
 
 ## Development
 
-### Gulp tasks
+### Test commands
 
-- `gulp watch` executes the unit tests in watch mode (use `gulp test.unit` for a
+- `ibazel test test:unit_test` executes the unit tests in watch mode (use `bazel test test:unit_test` for a
   single run),
-- `gulp test.e2e` executes the e2e tests,
-- `guld test.golden` executes the golden tests,
+- `bazel test test:e2e_test` executes the e2e tests,
+- `bazel test test:golden_test` executes the golden tests,
 - `gulp test.check-format` checks the source code formatting using
   `clang-format`,
-- `gulp test` runs unit tests, e2e tests and checks the source code formatting.
+- `yarn test` runs unit tests, e2e tests and checks the source code formatting.
```

Comment:
does it work with the environment variables below?

Reply:
Now it does.